web/ts/watch.ts

```diff
@@ -126,6 +180,7 @@ export function createServerMessage(msg) {
 export function sendMessage(message: string, anonymous: boolean, replyTo: number) {
     ws.send(
         JSON.stringify({
+            type: "message",
```

Comment:
I think we can omit the `"` here, right?

Reply:
:+1: